MAINTAINERS.md

```diff
@@ -0,0 +1,377 @@
+GlusterFS Maintainers
```

Comment:
@amarts: Not sure whether I have made enough contributions to be in the maintainer list for last year